squash.md

```diff
@@ -201,35 +201,59 @@ the draw) for downloading required packages. If this happens on the initial
 deployment, it is recommended to destroy and recreate the instance.
 
 
-Initialize django database
+Initialize the QA database
 ---
+Note: this step must be executed if you want a fresh database and load some test data into it. Skip to the next section if you want to restore a copy of the latest production QA database.
 
     vagrant ssh squash
+
     cd /opt/apps/qa-dashboard
     . /opt/rh/rh-python35/enable
     . venv/bin/activate
     cd squash
 
-    export QA_USER="nobody"
-    export QA_USER_EMAIL=${USER}@example.com
     export DJANGO_SETTINGS_MODULE="squash.settings.production"
-
-    # XXX this is an ugly hack
-    # sudo -E -u squash python manage.py makemigrations
-    # running makemigrations under sudo fails:
-    # ImportError: No module named django.core.management
-    # despite VIRTUAL_ENV being preserved
-
-    # there does not appear to an option to change the migration file path
-    sudo chmod 777 /opt/apps/qa-dashboard/squash/dashboard/migrations/
-    python manage.py makemigrations
     python manage.py migrate
+    
+    export QA_USER="nobody"
+    export QA_USER_EMAIL=${USER}@example.com
     python manage.py createsuperuser --username $QA_USER --email $QA_USER_EMAIL --noinput
-    python manage.py loaddata initial_data
```

Comment:
Agreed. It is required for a 'new' deployment using the current production branch, but will change soon to the new command ```python manage.py loaddata test_data```.